### ISSUE TITLE
Upload speed is always 0 on gigabit link

### ISSUE THREAD

**Issue by claytondukes** [sivel/speedtest-cli#335]
Is this because my bandwidth is too high perhaps?
speedtest-cli has always worked for me, but I recently got Google Fiber (this week). Since then, all upload tests show 0. Is there a way I can fix this?

```
Retrieving speedtest.net configuration...
Retrieving speedtest.net server list...
Testing from Google Fiber (n.n.n.n)...
Selecting best server based on latency...
Hosted by Google Fiber (Raleigh, NC) [18.39 km]: 8.914 ms
Testing download speed............................................
Download: 880.18 Mbit/s
Testing upload speed..................................................
Upload: 0.00 Mbit/s
```

**Comment by sivel** [comment]
Please test with the recently released v1.0.0.

**Comment by claytondukes** [comment]
Works now, thanks!
